numpy/_core/code_generators/ufunc_docstrings.py

```diff
@@ -3619,8 +3619,9 @@ def add_newdoc(place, name, doc):
     Notes
     -----
     There is more than one definition of sign in common use for complex
-    numbers.  The definition used here is equivalent to :math:`x/\\sqrt{x*x}`
-    which is different from a common alternative, :math:`x/|x|`.
+    numbers.  The definition used here, :math:`x/|x|`, is different from a 
```

Comment:
Thanks and sorry that I missed that when I implemented the new definition. I think the previous definition was pretty uncommon, actually, so maybe write:
```
numbers.  The definition used here, :math:`x/|x|`, is the more common
and useful one, but is different from the one used in numpy prior to version 2.0,
:math:...
```

Reply:
Agreed, and pushed as suggested =)